frontend/src/pages/modelCatalog/ModelCatalogPage.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { ModelCatalogContext } from '~/concepts/modelCatalog/context/ModelCatalogContext';
+
+const ModelCatalogPage: React.FC = () => {
+  const { modelCatalogSources } = React.useContext(ModelCatalogContext);
+  const [sources, loaded, error] = modelCatalogSources;
+
+  React.useEffect(() => {
+    // eslint-disable-next-line no-console
+    console.log('Loaded:', loaded);
+  }, [sources, loaded, error]);
```

Comment:
You'll have to change how you consume this with the change to FetchStateObject - you'll just have `modelCatalogSources.data` (and `.loaded` and `.error`).